src/NuGet/Microsoft.Orleans.OrleansRuntime.nuspec

```diff
@@ -25,5 +25,8 @@
     <file src="OrleansRuntime.dll" target="lib\net45" />
     <file src="OrleansRuntime.pdb" target="lib\net45" />
     <file src="$SRC_DIR$OrleansRuntime\**\*.cs" target="src" />
+    <file src="OrleansDependencyInjection.dll" target="lib\net45" />
+    <file src="OrleansDependencyInjection.pdb" target="lib\net45" />
+    <file src="$SRC_DIR$OrleansDependencyInjection\*.cs" target="src" />
```

Comment:
This is required for pushing symbols NuGets to https://www.symbolsource.org/. The symbols and source files don't get included into normal NuGets. http://docs.nuget.org/Create/Creating-and-Publishing-a-Symbol-Package.